app/code/Magento/Payment/view/base/web/js/model/credit-card-validation/credit-card-number-validator/credit-card-type.js

```diff
@@ -103,7 +103,7 @@ define([
         {
             title: 'Maestro Domestic',
             type: 'MD',
-            pattern: '^6759(?!24|38|40|6[3-9]|70|76)|676770|676774\\d*$',
+            pattern: '^(6759(?!24|38|40|6[3-9]|70|76)|676770|676774)\\d*$',
```

Comment:
Can you add a test?

Reply:
There wasn't existing coverage of this type validator, but I added a check for it on `Magento_Payment/js/model/credit-card-validation/validator`.

I'm new to it, so pardon any broken conventions.